File: app/src/main/java/io/plan8/backoffice/model/api/Team.java
package io.plan8.backoffice.model.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.plan8.backoffice.model.BaseModel;

/**
 * Created by chokwanghwan on 2017. 11. 28..
 */

public class Team implements BaseModel {
    @SerializedName("id")
    @Expose()
    int teamId;
    @SerializedName("publicId")
    @Expose()
    String publicId;
    @SerializedName("name")
    @Expose()
    String name;
    @SerializedName("logo")
    @Expose()
    String logo;
    @SerializedName("configuration")
    @Expose()
    Configuration configuration;
    @SerializedName("ownerId")
    @Expose()
    String ownerId;
    @SerializedName("deactivated")
    @Expose()
    boolean deactivated;

    public Team() {
    }

    public int getTeamId() {
        return teamId;
    }

    public String getPublicId() {
        return publicId;
    }

    public String getName() {
        return name;
    }

    public String getLogo() {
        return logo;
    }

    public Configuration getConfiguration() {
        return configuration;
    }

    public String getOwnerId() {
        return ownerId;
    }

    public boolean isDeactivated() {
        return deactivated;
    }
}

File: app/src/main/java/io/plan8/backoffice/model/api/Team.kt
package io.plan8.backoffice.model.api

import com.google.gson.annotations.SerializedName
import io.plan8.backoffice.model.BaseModel

/**
 * Created by SSozi on 2017. 11. 20..
 */
class Team: BaseModel {
    @SerializedName("id") lateinit var teamId: String
    @SerializedName("name") lateinit var teamName: String
    @SerializedName("logo") lateinit var teamLogo: String
    @SerializedName("configuration") lateinit var configuration: Configuration
    @SerializedName("ownerId") lateinit var ownerId: String
    @SerializedName("deactivated") var deactivated: Boolean = true

    //TODO : member, admin 추가해야함
}